src/app/login/page.js

```diff
@@ -1,11 +1,108 @@
 "use client";
+import React, { useState } from "react";
+import { Form, Button, Alert } from "react-bootstrap";
+
+const Login = () => {
+  const [inputUsername, setInputUsername] = useState("");
+  const [inputPassword, setInputPassword] = useState("");
+
+  const [show, setShow] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    setLoading(true);
+    await delay(500);
+    console.log(`Username :${inputUsername}, Password :${inputPassword}`);
+    if (inputUsername !== "admin" || inputPassword !== "admin") {
+      setShow(true);
+    }
+    setLoading(false);
+  };
+
+  const handlePassword = () => {};
+
+  function delay(ms) {
+    return new Promise((resolve) => setTimeout(resolve, ms));
+  }
 
-const page = () => {
   return (
-    <div>
-    <h2>This will be the login page.</h2>
+    <div
+      className="sign-in__wrapper"
+      style={{ backgroundImage: `url(${BackgroundImage})` }}
```

Comment:
Line 32, BackgroundImage is not defined. This is throwing an error when trying to see the form.